.github/workflows/next-binaries.yml

```diff
@@ -0,0 +1,65 @@
+name: Publish Nightly Next Binaries
+on:
+  schedule:
+    - cron: '10 3 * * *'
+  workflow_dispatch:
+    name:
+      description: 'Name of the ouput file'
+      required: false
+      default: ''
+
+# This workflow is mostly a copy of publish binaries
+# This is by choice as the next branch will eventually be merged in the master
+# It makes it easy to remove when the time comes
+jobs:
+  pre_release_bin:
+    name: Build and Publish Binaries
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@next
```

Comment:
Why use next and not master here?

Reply:
That's the whole purpose of this pr

Reply:
See: https://github.com/actions/checkout#checkout-a-different-branch

I think what you want to do here is:

```
- uses: actions/checkout@v2
  with:
    ref: next
```